Exception in thread "Thread-1" java.lang.RuntimeException: java.net.BindException: Address already in use (Bind failed)
	at server.UdpServerThread.run(UdpServerThread.java:31)
Caused by: java.net.BindException: Address already in use (Bind failed)
	at java.net.PlainDatagramSocketImpl.bind0(Native Method)
	at java.net.AbstractPlainDatagramSocketImpl.bind(AbstractPlainDatagramSocketImpl.java:93)
	at java.net.DatagramSocket.bind(DatagramSocket.java:392)
	at java.net.DatagramSocket.<init>(DatagramSocket.java:242)
	at java.net.DatagramSocket.<init>(DatagramSocket.java:299)
	at java.net.DatagramSocket.<init>(DNov 25, 2022 4:49:29 PM server.Server main
INFO: test2
Nov 25, 2022 4:49:29 PM server.CityReservationSystem reserveTicket
SEVERE: You cannot reserve more than one event of the same type in the same day
